pkgs/top-level/all-packages.nix

```diff
@@ -32867,6 +32776,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) AppKit Foundation;
   };
 
+  wrapRetroArch = { retroarch }: callPackage ../misc/emulators/retroarch/wrapper.nix {
+    inherit retroarch;
+  };
+
   retroarch = wrapRetroArch { retroarch = retroarchBare; };
```

Comment:
Maybe we could also include a `retroarchFull` derivation that is `retroarch` + all available `libretro` cores. This would make it easier for newbies who are starting on Nix on how to get a working RetroArch setup.

WDYT :thinking: ?